internal/daysteps/daysteps.go

```diff
@@ -1,15 +1,40 @@
 package daysteps
 
 import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
 	"time"
+
+	"github.com/Yandex-Practicum/go1fl-4-sprint-final/internal/spentcalories"
 )
 
 var (
-	StepLength = 0.65 // длина шага в метрах
+	StepLength    = 0.65  // длина шага в метрах
+	distance      float64 //пройденная дистанция
+	ErrFormatTime = errors.New("Ошибка преобразования строки в формат времени")
+	ErrConvToInt  = errors.New("ошибка преобразования в целое число")
+	ErrLenSlice   = errors.New("len(word) < 2")
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// ваш код ниже
+	// Строка в слайс строк
+	word := strings.Split(data, ",")
+	if len(word) < 2 {
+		return 0, 0, ErrLenSlice
+	}
+	//преобразование строки в целое число
+	step, err := strconv.Atoi(word[0])
+	if err != nil {
+		return 0, 0, ErrConvToInt
+	}
+	//Парсим время
+	duration, err := time.ParseDuration(word[1])
+	if err != nil {
+		return 0, 0, ErrFormatTime
```

Comment:
1. Теряешь сообщение об ошибки
2. Ошибку лучше писать на английском
3. Ошибку лучше начинать с маленькой буквы
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```

Reply:
Нужно везде поправить